src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -322,7 +327,7 @@ Status CoreWorkerMemoryStore::GetImpl(const std::vector<ObjectID> &object_ids,
     }
 
     // Return if all the objects are obtained.
```

Comment:
...", or any existing objects are known to have exception."

Reply:
done